code/modules/resleeving/computers.dm

```diff
@@ -88,9 +89,26 @@
 		menu = 4
 		to_chat(user, "<span class='notice'>\The [src] loads the body record from \the [W] before ejecting it.</span>")
 		attack_hand(user)
+	if(istype(W, /obj/item/implant/mirror))
+		user.drop_item()
+		W.forceMove(src)
+		hasmirror = W
+		user.visible_message("[user] inserts the [W] into the [src].", "You insert the [W] into the [src].")
+	return ..()
+
+/obj/machinery/computer/transhuman/resleeving/verb/eject_mirror()
+	set category = "Object"
+	set name = "Eject Mirror"
+	set src in oview(1)
+
+	if(hasmirror)
+		to_chat(usr, "You eject the mirror.")
+		(!usr.get_active_hand() && istype(usr,/mob/living/carbon/human))
```

Comment:
?

Reply:
It checks if they have an open hand, and if the person doing it is a human which will have a hand to put the item in